mycroft/client/speech/main.py

```diff
@@ -35,14 +35,14 @@
 config = ConfigurationManager.get()
 
 
-def handle_listening():
-    logger.info("Listening...")
-    client.emit(Message('recognizer_loop:listening'))
+def handle_record_begin():
+    logger.info("Begin Recording...")
+    client.emit(Message('recognizer_loop:record_begin'))
```

Comment:
By emitting this event in the handler, of this event, you are creating a message storm that will eventually bring the recognizer loop to a halt. You do the same thing the record_end event handler. This is the cause of the DDOS that @the7erm is talking about.